### PR TITLE
Improve working-with-gatekeeper.md

### PR DESCRIPTION
**Pull request by AllenZMC**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Modify the docs for a little readability

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: docs/working-with-gatekeeper.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
 # Working with Gatekeeper(OPA)
 
-[Gatekeeper](https://github.com/open-policy-agent/gatekeeper) , is a customizable admission webhook for Kubernetes that enforces policies executed by the Open Policy Agent (OPA), a policy engine for Cloud Native environments hosted by [Cloud Native Computing Foundation ](https://cncf.io/).
+[Gatekeeper](https://github.com/open-policy-agent/gatekeeper) , is a customizable admission webhook for Kubernetes that enforces policies executed by the Open Policy Agent (OPA), a policy engine for Cloud Native environments hosted by [Cloud Native Computing Foundation](https://cncf.io/).
 
-This document gives an example to demonstrate how to use the `Gatekeeper` to manage OPA policy. 
+This document provides an example to demonstrate how to use the `Gatekeeper` to manage OPA policy.
```
Comment (RainbowMango) [inline docs/working-with-gatekeeper.md:5]:
```suggestion
This document demonstrates how to use the `Gatekeeper` to manage OPA policy. 
```
@Tingtal How about this?

Comment (Tingtal) [reply docs/working-with-gatekeeper.md:5]: Nice change! Another small touch: 
```suggestion
This document demonstrates how to use the `Gatekeeper` to manage OPA policies. 
```

Comment (AllenZMC) [reply docs/working-with-gatekeeper.md]: @RainbowMango Updated.

**Review by RainbowMango — APPROVED** [review]
/lgtm
/approve

**Pull request merged**